src/DTO/Address.php

```diff
@@ -27,6 +28,7 @@ class Address extends DTO implements JsonSerializable, DTOInterface
 {
     public ?string $address1;
     public ?string $address2;
+    public ?string $address_code;
```

Comment:
I think this is in the wrong place. It will mean all addresses have address code.
`ChannelOrder.shipping_address` and `ChannelOrder.billing_address` don't have address code.
You need to add address code to `SystemCustomerAddress`